packages/ui/src/components/DateInput/constants.ts

```diff
@@ -0,0 +1,48 @@
+export const POPUP_WIDTH = '264px'
```

Comment:
We use rem now so you can set this:

```suggestion
export const POPUP_WIDTH = '16.5rem'
```